Yeta-Wrench.md

```diff
@@ -1 +1,14 @@
-![](http://loenwind.info/eio/Yeta_Wrench.png)
\ No newline at end of file
+# Yeta Wrench
+![](http://loenwind.info/eio/Yeta_Wrench.png)
+
+The Yeta Wrench is a tool that has a lot of uses:
+
+* Conduit Facades are hidden while the Yeta Wrench is held
+
+* Machines and Conduits can be broken by Shift + Right Clicking with the Yeta Wrench
+
+* Extract and Insert(or Input and Output) modes can be changed by Right Clicking with the Yeta Wrench on a Conduit Bundle
+
+* Conduits can be prevented from connecting by Right Clicking with the Yeta Wrench on the connection
```

Comment:
"prevented" isn't quite right. they can be connected and disconnected from each other and from blocks/machines

Reply:
Disconnected would probably work